src/data/daysAndTimeSlots.js

```diff
@@ -54,6 +54,12 @@ export const timeOptions = [
   },
 ];
 
+export const roomOptions = [
+  { value: "All Classrooms", label: "All Classrooms" },
+  { value: "Regular Classrooms", label: "Regular Classrooms" },
+  { value: "Lab Classrooms", label: "Lab Classrooms" },
```

Comment:
The `value` and `label` should not be same, the labels are good in your code. 
But for programmatic jobs we will use the `value` field, it's easier if that field is following `camelCasing` or `snake_casing` convention.